src/main/java/IP/duke/main/Parser.java

```diff
@@ -0,0 +1,54 @@
+package IP.duke.main;
+
+import IP.duke.command.AddCommand;
+import IP.duke.command.Command;
+import IP.duke.command.DeleteCommand;
+import IP.duke.command.ExitCommand;
+import IP.duke.command.FilterCommand;
+import IP.duke.command.ListCommand;
+import IP.duke.command.MarkDoneCommand;
+
+/**
+ * Represents a abstraction to read user input.
+ * 
+ * @author Gordon Yit
+ * @version CS2103T, Semester 2
+ */
+public class Parser {
+    
+    public enum Commands { done, list, delete, filter, others};
```

Comment:
Should the brackets be in Egyptian style and without the semicolon?
```suggestion
    public enum Commands { 
        done, list, delete, filter, others
    }
```